scripts/clean-db-dump.sql

```diff
@@ -112,6 +112,36 @@ WHERE declaration IS NOT NULL
   AND declaration::varchar != 'null'
   AND declaration::varchar != '{}';
 
+-- Create some fake draft and closed briefs using eligible suppliers
+INSERT INTO brief_responses (
+      data,
+      brief_id,
+      supplier_id,
+      created_at,
+      submitted_at
+)
+SELECT
+    '{"essentialRequirements": [{"evidence": "They have the correct number of hooves."}, {"evidence": "So shiny..."}, {"evidence": "All at least 50"}], "niceToHaveRequirements": [{"yesNo": false}, {"yesNo": true, "evidence": "No jump is too high."}, {"yesNo": true, "evidence": "NEIGH"}], "availability": "09/09/17", "essentialRequirementsMet": true, "respondToEmailAddress": "example-email@gov.uk"}'::json,
+    eligible_brief_supplier_pairings.brief_id,
+    eligible_brief_supplier_pairings.supplier_id,
+    now(),
+    CASE
+      WHEN MOD(id, 5) = 0 THEN NULL
+      ELSE now()
+    END
+FROM (
+    SELECT DISTINCT ON (briefs.id)
+      briefs.id AS brief_id,
+      supplier_id as supplier_id
+    FROM supplier_frameworks
+      LEFT JOIN frameworks ON supplier_frameworks.framework_id = frameworks.id
+      LEFT JOIN briefs ON briefs.framework_id = frameworks.id
+    WHERE declaration->>'status' = 'complete'
+      AND declaration->>'organisationSize' != ''
```

Comment:
I'm not too up to date with services, but is there ever a reason why the declaration would be complete but they don't have an organisation size?

Reply:
Copied declarations should all have a value here - see https://github.com/alphagov/digitalmarketplace-aws/blob/master/scripts/clean-db-dump.sql#L108

Reply:
Live also all have it too as it's been a required value on all declarations.

Reply:
Found a way. In terms of creating test data I think it'll do. Creates an `essentialRequirements` with the same number of 'evidences' as there are essential requirements.